go.mod

```diff
@@ -0,0 +1,78 @@
+module knative.dev/pkg
```

Comment:
will this than be also done for all the oter knative repos ?

Reply:
The finally goal will be switching all Knative projects to use Go modules. In case you have concerns or questions, please join #productivity for discussions :)